lib/index.js

```diff
@@ -113,4 +113,19 @@ function returnParsedResponse(response) {
 	return response.parsedBody || response.parsedText || response;
 }
 
-module.exports = Fetcher;
\ No newline at end of file
+module.exports = Fetcher;
+
+function isJsonContentType (contentType) {
```

Comment:
Maybe a few tests for this would be helpful, also when someone else has to edit it eventually